pages/linux/systemd-run.md

```diff
@@ -0,0 +1,28 @@
+# systemd-run
+
+> Run programs in transient scope units, service units, or path-, socket-, or timer-triggered service units.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-run.html>.
+
+- Run a program as root:
+
+`systemd-run {{executable}}`
```

Comment:
```suggestion
`systemd-run {{path/to/executable}}`
```
Question to other maintainers: Would this be applicable here, or is the current one fine?

Reply:
I took a look around, and I feel like commands that resolve `PATH`, for example `which` and `gdb`, use the form I used, while the ones that don't, for example `journalctl` or `ldd`, use your form. `systemd-run` does resolve `PATH`, so I feel like the current form fits better.

Reply:
To make it [convertible](https://github.com/command-line-interface-pages/v2-tooling/tree/main/md-to-clip) to Command Line Interface Page's placeholder it should contain `path/to`.

Reply:
@EmilySeville7cfg, I have no idea what that means :stuck_out_tongue: So, what should I do?